floris/simulation/flow_field.py

```diff
@@ -34,7 +34,8 @@ class FlowField(FromDictMixin):
     wind_shear: float = field(converter=float)
     air_density: float = field(converter=float)
     turbulence_intensity: float = field(converter=float)
-    reference_wind_height: float = field(converter=float)
+    reference_wind_height: int = field(converter=int)
```

Comment:
I'm guessing this is by mistake. I think we want to keep `reference_wind_height` a float

Reply:
Yes good catch @Bartdoekemeijer